### ISSUE TITLE
fix SVG output

### ISSUE THREAD

**Issue by pkra** [krautzource/mathjax-sre-walker#39]
Regression from #37. The SVG output no longer works (at least in Chrome+NVDA).
 

* Since MathJax v3's SVG output has changed where the root ends up (cf https://github.com/krautzource/mathjax-sre-walker/issues/37#issuecomment-525781636 and the change in CSS), we no longer have a label on the SVG element.
* navigation no longer voices the activedescendant

**Comment by pkra** [comment]
While I'd think it should be possible to make the svg element "inert" (add role presentation or group, strip the dated MathJax output markup etc) it seems NVDA+Chrome doesn't like it too well.

So as an alternative, we can move the necessary attributes to the svg and attach the navigator to the svg. A bit of a hack but should bring things back to the way they were.
